python/mxnet/ndarray/sparse.py

```diff
@@ -883,6 +884,59 @@ def _ndarray_cls(handle, writable=True, stype=_STORAGE_TYPE_UNDEFINED):
 _set_ndarray_class(_ndarray_cls)
 
 
+def eye(stype, N, M=0, k=0, ctx=None, dtype=None, aux_types=None, **kwargs):
+    """Return Return a 2-D array with ones on the diagonal and zeros elsewhere
+       of given shape and type.
+
+    Parameters
+    ----------
+    stype: string
+        The storage type of the return array, such as 'row_sparse', 'csr', etc
+    N: int
+        Number of rows in the output.
+    M : int, optional
+        Number of columns in the output. If 0, defaults to N.
+    k : int, optional
+        Index of the diagonal: 0 (the default) refers to the main diagonal,
+        a positive value refers to an upper diagonal,
+        and a negative value to a lower diagonal.
+    ctx : Context, optional
+        An optional device context (default is the current default context)
+    dtype : str or numpy.dtype, optional
+        An optional value type (default is `float32`)
+    aux_types: list of numpy.dtype, optional
```

Comment:
aux_types will be removed by https://github.com/apache/incubator-mxnet/pull/8269
let's also not expose aux_types to users here, either. just using the default one is fine

Reply:
Done.